packages/communication-react/rollup.config.js

```diff
@@ -3,15 +3,27 @@
 
 import Package from './package.json';
 import commonConfig from '../../common/config/rollup/rollup.config';
+import commonjs from '@rollup/plugin-commonjs';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import svg from 'rollup-plugin-svg';
+import json from '@rollup/plugin-json';
 
 export default [
   {
     ...commonConfig(Package),
     input: './dist/dist-esm/communication-react/src/index.js',
     output: {
-      file: './dist/dist-cjs/communication-react/index.js',
+      dir: './dist/dist-cjs/communication-react',
       format: 'cjs',
       sourcemap: true
-    }
+    },
+    plugins: [
+      commonjs(),
```

Comment:
The common rollup config has `[commonjs(), sourcemaps(), svg()]` (`common\config\rollup\rollup.config.js`), can the json one be added to the common config and these removed from this file?